tip-calculator/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tip Calculator</title>
+    <link href="https://fonts.googleapis.com/css?family=Jura&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <h1>Hack your tips!</h1>
+    <p>Rather than try and figure out what kind of tip to leave using a percentage, just rate your experience!</p>
+    <div class="app">
+        <div class="calculator">
+            <header>
+                <h1>Tip Hacker</h1>
+                <form>
+                    <h3>What's the bill?</h3>
+                    <input id="bill" type="text" placeholder="Let's hack it!">
+                    <h3>How was this experience for you?
+                        <select id="rating">
+                            <option>Be real, how was it?</option>
+                            <option value="0.0">0% - I should have stayed home.</option>
```

Comment:
instead of adding these values in the HTML you can store the data in the index.js file and use jquery to add them to the select tag.